must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -195,21 +195,22 @@ for ns in $namespaces; do
     nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
 
     # Collecting ceph prepare volume logs
-    for node in ${nodes}; do
-        printf "collecting prepare volume logs from node %s \n"  "${node}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
-        NODE_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/namespaces/${ns}/osd_prepare_volume_logs/${node}
-        mkdir -p "${NODE_OUTPUT_DIR}"
-        { timeout 120 oc debug nodes/"${node}" -- bash -c "test -f /host/var/lib/rook/log/${ns}/ceph-volume.log && cat /host/var/lib/rook/log/${ns}/ceph-volume.log" > "${NODE_OUTPUT_DIR}"/ceph-volume.log; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
-    done
+    volume_collection(){
+        printf "collecting prepare volume logs from node %s \n"  "${node}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/log "${NODE_OUTPUT_DIR}"
+    }
 
     for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
         { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
     done
 
     oc delete -f pod_helper.yaml
 
+    # creating a counter variable for collecting PID in array
+    idx=0
     crash_collection(){
-        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 8m" &
```

Comment:
any specific reason for increase in sleep.

Reply:
sure,fxed